packages/api-gateway/src/drivers/http/adapters/places/handlers.js

```diff
@@ -11,9 +11,30 @@ async function postPlace(req, reply) {
 
   return reply.code(201)
     .header('Content-Type', 'application/json; chartset:utf-8')
```

Comment:
nit-pick: I think that you don't need to specify the content type, fastify will automatically set it for you, can you remove this line?